sap_hana/README.md

```diff
@@ -72,6 +72,24 @@ To learn how to set the port number for HANA tenant, single-tenant, and system d
    GRANT DD_MONITOR TO <USER>;
    ```
 
+#### Dependencies
+
+Although not required, it is recommended that the [hdbcli][10] client library be installed since it is the only officially maintained library and supports many new features.
```

Comment:
```suggestion
Although not required, it is recommended to install the [hdbcli][10] client library since it is the only officially maintained library and supports many new features.
```